inc/body/body.php

```diff
@@ -1,5 +1,5 @@
 <?php
-include 'inc/body/google-tag-manager.inc';
-include 'inc/body/tna-header.inc';
-include 'inc/body/tna-nav.php';
+include '../inc/body/google-tag-manager.inc';
```

Comment:
I think that this should just have the file names. For example it would just be "google-tag-manager.inc" and not "../inc/body/google-tag-manager.inc" for all the includes in this file